client/test/role.go

```diff
@@ -310,7 +310,9 @@ func (r *role) GoHandle(rng *rand.Rand) (h *acceptNextPropHandler, wait func())
 	}
 }
 
-const challengeDuration = 60
+// Must be large since the simulated backend increases the block time in auto
+// mining by 10s per block and 100 seconds/second in total.
+const challengeDuration = 6000
```

Comment:
I agree that the documentation is misleading here.  
The `blockInterval` is an implementation detail of the Etheruem backend. This file here in `client/test` does not know about Ethereum. `blockInterval` is also not exported…    
In the future it will have to work with multiple blockchain backends, the easiest solution would be to fix it to a very large value.  
What do you think @matthiasgeihs ?

Reply:
OK, in this case this const should definitely not depend on an implementation detail of the Ethereum simulated backend.

The right approach, I think, is to declare the challenge duration depending on its context. This means we cannot use the same challengeDuration for all tests. I created a respective commit: https://github.com/perun-network/go-perun/commit/d9532b2a5362e904966d2de35fb0e3852aec8c0c

Reply:
Ok that is the cleanest solution, will cherry pick the commit, thanks.